buddychatcli/Program.cs

```diff
@@ -19,7 +21,75 @@ static int Main(string[] args)
 
             Console.WriteLine("Command: " + options.Command);
 
+            createParticipantData(SignUpFileName);
+
+            
             return 0;
         }
+
+        // Creating Participant Data using Sign up sheet and new signup information
+        private static void createParticipantData (string filename) 
+        {

```

Comment:
I'd recommend separating logic that parses data from reading in data. It'll make your parsing function easier to test which can be a good way to quickly validate the function :)